nikola/post.py

```diff
@@ -1085,11 +1085,13 @@ def previewimage(self, lang=None):
 
         image_path = self.meta[lang]['previewimage']
         if not image_path:
-            return self._default_preview_image
+            image_path = self._default_preview_image
 
-        # This is further parsed by the template, because we don’t have access
-        # to the URL replacer here.  (Issue #1473)
-        return image_path
+        if image_path.startswith("/"):
```

Comment:
Changing this line to `if image_path and image_path.startswith("/"):` works for me. Otherwise I get an exception.

```python
    if image_path.startswith("/"):
AttributeError: 'NoneType' object has no attribute 'startswith'
```

Reply:
Ah but your CI is [already telling you this](https://travis-ci.org/github/getnikola/nikola/jobs/742854436#L679) ;)

Reply:
I added a slightly different fix, seems to work now in all cases (and I forgot to look at CI, assuming it’s an expected failure of baseline testing). Can you test again?

Reply:
Seems to work, cheers.

Reply:
I'd myself have gone for an explicit `image_path is None` rather than `not image_path` ;)

Reply:
For `image_path = ""`, returning a blank screen is also fine.

Reply:
Ah good point. I only had my case in mind :)